js/index.js

```diff
@@ -63,6 +63,103 @@ function showCopyright() {
   copyright.innerText = `© Saul Castillo | ${thisYear}`;
   //7. Append the copyright paragraph into the footerNav element.
   footerNav.appendChild(copyright);
+  //8. Appending social media links
+  /*
+    <address>
+      <ul class="horizontal-list">
+              <li class="connect-item">
+                  <a href="mailto:secch97@gmail.com" target="_blank">
+                      <i class="fa-solid fa-envelope"></i>
+                      Mail
+                  </a>
+              </li>
+              <li class="connect-item">
+                  <a href="https://github.com/secch97" target="_blank">
+                      <i class="fa-brands fa-github"></i>
+                      GitHub
+                  </a>
+              </li>
+              <li class="connect-item">
+                  <a href="https://twitter.com/MrRobot_97" target="_blank">
+                      <i class="fa-brands fa-twitter"></i>
+                      Twitter
+                  </a>
+              </li>
+      </ul>
+    </address>
+  */
+  //8.1 Creating and setting up HTML elements
+  //8.1.1 Creating the address element
+  const socialMediaListContainer = document.createElement("address");
+  //8.1.2 Creating the list element
+  const socialMediaList = document.createElement("ul");
+  //8.1.3 Setting up HTML elements
+  socialMediaList.classList.add("horizontal-list");
+  socialMediaListContainer.classList.add("footer-element")
+
+  //8.2 Setting up and getting a fetch request with the social media data:
+  
+  //8.2.1 Set up:
+
+  //8.2.1.1 Variables:
+  const method = "GET";
+  const url = "./js/socialMedia.json";
```

Comment:
None of the json files are loading (seeing console errors regarding CORS). I think since these chunks of data are so small, it would be best if they were just variables here in our js file, which would save us a lot of code complexity and debugging.

Reply:
You are right, these json files I created have small chunk of data compared to the github repos JSON.

P.D: I am running my web page with VSCode's Live Server and the JSON loaded as expected. Since you said they were not loading, I checked running my page locally by opening the index.html on chrome: just like you said, the JSON were not loading. I removed the data fetching and turned it into variables like you said and now it loads correctly!